File: payroll-service/Employee/EmployeeModel.vb
Imports MySql.Data.MySqlClient

Namespace Employee
    Public Class EmployeeModel

        Public Property EE_Id As String = ""
        Public Property First_Name As String = ""
        Public Property Last_Name As String = ""
        Public Property Middle_Name As String = ""
        Public Property Location As String = ""
        Public Property TIN As String = ""

        Public Property Card_Number As String = ""
        Public Property Account_Number As String = ""
        Public Property Payroll_Code As String = ""
        Public Property Bank_Category As String = ""
        Public Property Bank_Name As String = ""

        Public Date_Modified As Date

        Public ReadOnly Property Fullname As String
            Get
                Return String.Format("{0},{1} {2}", Last_Name, First_Name, IIf(Middle_Name <> "", Middle_Name(0) & ".", ""))
            End Get
        End Property

        Sub New()

        End Sub

        Sub New(reader As MySqlDataReader)
            EE_Id = reader.Item("ee_id")
            First_Name = reader.Item("first_name")
            Last_Name = reader.Item("last_name")
            Middle_Name = reader.Item("middle_name")
            Account_Number = reader.Item("account_number")
            Card_Number = reader.Item("card_number")
            Bank_Category = reader.Item("bank_category")
            Bank_Name = reader.Item("bank_name")
            Payroll_Code = reader.Item("payroll_code")
            Location = reader.Item("location")
            TIN = reader.Item("tin")
            Date_Modified = reader.Item("date_modified")
        End Sub
    End Class

End Namespace

File: employee-module/EmployeeModel.cs
using MySql.Data.MySqlClient;
using System;

namespace employee_module
{
    public class EmployeeModel
    {
        public string EE_Id { get; set; }
        public string First_Name { get; set; }
        public string Last_Name { get; set; }
        public string Middle_Name { get; set; }
        public string Location { get; set; }

        public string Card_Number { get; set; }
        public string Account_Number { get; set; }
        public string Payroll_Code { get; set; }
        public string Bank_Category { get; set; }
        public string Bank_Name { get; set; }

        public string Pagibig { get; set; }
        public string PhilHealth { get; set; }
        public string SSS { get; set; }
        public string TIN { get; set; }

        public DateTime Date_Modified { get; set; }

        public string Fullname
        {
            get
            {
                if (First_Name is null || Last_Name is null) { return ""; }
                string _fullName = $"{Last_Name}, {First_Name}";
                if (Middle_Name == "" || Middle_Name is null) { _fullName = $"{_fullName}."; } else { _fullName = $"{_fullName} {Middle_Name.Substring(0, 1)}."; }
                return _fullName;
            }
        }

        public EmployeeModel() { }

        public EmployeeModel(MySqlDataReader reader)
        {

            EE_Id = reader.GetString("ee_id");
            First_Name = reader.GetString("first_name");
            Last_Name = reader.GetString("last_name");
            Middle_Name = reader.GetString("middle_name");
            Account_Number = reader.GetString("account_number");
            Card_Number = reader.GetString("card_number");
            Bank_Category = reader.GetString("bank_category");
            Bank_Name = reader.GetString("bank_name");
            Payroll_Code = reader.GetString("payroll_code");
            Location = reader.GetString("location");
            TIN = reader.GetString("tin");
            Pagibig = reader.GetString("pagibig");
            SSS = reader.GetString("sss");
            PhilHealth = reader.GetString("philhealth");

            Date_Modified = reader.GetDateTime("date_modified");
        }
    }
}
